lighthouse-core/report/v2/renderer/category-renderer.js

```diff
@@ -233,21 +233,23 @@ class CategoryRenderer {
     element.id = category.id;
     element.appendChild(this._renderCategoryScore(category));
 
-    const auditsGroupedByGroup = category.audits.reduce((indexed, audit) => {
+    const auditsGroupedByGroup = /** @type {!Object<string,
+        {passed: !Array<!ReportRenderer.AuditJSON>,
+        failed: !Array<!ReportRenderer.AuditJSON>}>} */ ({});
```

Comment:
open to less awkward type cast here, but not sure how. One option is to split `auditsGroupedByGroup` into two `!Object<string, !Array<!ReportRenderer.AuditJSON>>`, one for passes and one for failed, but wanted to minimize change to @patrickhulce's code